orchestrators/QuotationIntelligence/tests/Feature/EndToEndQuoteProcessingTest.php

```
@@ -507,4 +508,81 @@ public function test_processes_with_empty_fx_lock_date_and_skips_invalid_lines()
+        );
+
+        $result = $coordinator->processQuote('tenant-1', 'doc-123');
+
+        $this->assertCount(1, $result['lines']);
+        $this->assertSame(1.0, $result['lines'][0]['normalized_quantity']);
+        $this->assertSame(0.6, $result['lines'][0]['ai_confidence']);
+        $this->assertSame(
+            'uom_conversion_failed',
+            $result['lines'][0]['metadata']['normalization_warnings'][0]['code']
+        );
+    }
```

PHPMD (2.15.0)

[error] The method test_preserves_line_and_degrades_confidence_when_uom_normalization_fails is not named in camelCase. (undefined)

(CamelCaseMethodName)